pilot/pkg/model/jwks_resolver.go

```diff
@@ -50,14 +50,14 @@ const (
 
 	// JwtPubKeyExpireDuration is the expire duration for JWT public key in the cache.
 	// After this duration expire, refresher job will fetch key for the cached item again.
-	JwtPubKeyExpireDuration = time.Hour
+	JwtPubKeyExpireDuration = time.Minute * 5
 
 	// JwtPubKeyEvictionDuration is the life duration for cached item.
 	// Cached item will be removed from the cache if it hasn't been used longer than JwtPubKeyEvictionDuration.
 	JwtPubKeyEvictionDuration = 24 * 7 * time.Hour
 
 	// JwtPubKeyRefreshInterval is the running interval of JWT pubKey refresh job.
-	JwtPubKeyRefreshInterval = time.Minute * 20
+	JwtPubKeyRefreshInterval = time.Minute * 5
```

Comment:
Same - even 20 min seemed to low, JWT pub key don't rotate every 20 min. 
Let the user customize with an env ( no need for a full flag ), and keep the larger default.